lua/swampshop/cl_draw.lua

```diff
@@ -132,6 +136,18 @@ function SS_ApplyBoneMods(ent, mods)
                     pso = pso + psn
                     ent:ManipulateBonePosition(x, pso)
                 end
+                if (item.configurable or {}).ang then
+                    local psn = cfg["ang" .. suffix] or Angle()
+
+                    --don't allow moving the root bone
+                    if ent:GetBoneParent(x) == -1 then
+
```

Comment:
Are you supposed to return here?

Reply:
oh yeah it looks like i took it out while testing